week-03/main.js

```diff
@@ -3,7 +3,8 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
-
+list.shift();
+list.push(1);
```

Comment:
While you are pushing `1` correctly into the tail end of the array, it's not really the same `1` that you shifted off the end. How would you capture that element?